pages/common/fd.md

```diff
@@ -0,0 +1,23 @@
+# fd
+
+> A simple, fast and user-friendly alternative to find.
+
+- Find files under current dir that match a pattern:
```

Comment:
Not sure, but it might be a good idea to expand `dir` to directory. What do you think, @agnivade / @waldyrious?